### DISCUSSION TITLE
ManageStyleSheets error on _Host.cshtml file

### DISCUSSION THREAD

**Discussion by LuisVilaplana** [oqtane/oqtane.framework#3484] [General] [answered]
Hello, I installed Oqtane 5.0 and got an error on _Host.cshtml file:

Line: 191 - ManageStyleSheets(resources, alias, theme.ThemeName);  ---- Object reference not set to an instance of an object.'

The theme object is null, it seems there is an error an line 177:

var theme = themes.FirstOrDefault(item => item.Themes.Any(item => item.TypeName == page.ThemeType));

So for now, to make it work I replaced  it with: 

var theme = themes.FirstOrDefault();

I got the same error upgrading an existing installation and on a brand new one.

Thansk

**Comment by sbwalker** [comment]
@LuisVilaplana which database are you using? I have done many installations and upgrades in the past 2 weeks without encountering any issues so I would like to determine the cause of your issue.

**Reply by LuisVilaplana** [reply, marked as answer]
Hi Shaun, no no, it was my mistake, I'm using Bootswash themes that I forgot to recompile for .net8. I already did it and the error disappeared. 

By the way, I edited the .cmd and .nuspec files and replaced  net7.0 to net8.0 in all file paths.

Thank you so much
